### ISSUE TITLE
TEST_File

### ISSUE THREAD

**Issue by JAMON96** [BPYap/BERTSel#2]
Hello, when you are processing the test set, did you only keep the questions with the correct answers and then do the mapping? In addition, is the attention_mask in your code the masking task in the BERT pre-training task?
![image](https://user-images.githubusercontent.com/42796458/127089303-a0dd54ca-9c6a-4b34-9f0f-671a5196b612.png)

**Comment by BPYap** [comment]
Hi, each line in the `test.tsv` file should contain a mapping from a question to a list of comma-delimited integers that corresponds to the correct answers stored in another text file (i.e. the answer pool). For example, this line `<question_1> 0, 2` means the first and the third line (index starts counting from 0) in the answer pool are the correct answers for `<question_1>`.

The `attention_mask` is used to zero out the embedding values for the padding tokens so that the self-attention mechanism won't attend to them during training.
